res/values-nn/strings.xml

```diff
@@ -298,5 +298,53 @@
     <string name="WarnSSLTitle">Åtvaring: SSL sertifikat</string>
     <string name="WarnSSLDetail">SSL sertifikatet til serveren er ikkje lista som tiltrudd. 
         Du kan framleis velge å fortsette eller avbryte førespurnaden.</string>
+    <string name="LinkToApp_Amazon">http://www.amazon.com/gp/mas/dl/android?p=com.duckduckgo.mobile.android</string>
+    <string name="LinkToApp_Google">https://play.google.com/store/apps/details?id=com.duckduckgo.mobile.android</string>
+    <string name="LinkToApp_GoogleStore">market://details?id=com.duckduckgo.mobile.android</string>
+    <string name="SearchStringBig">Søk DuckDuckGo eller skriv URL</string>
+    <string name="SearchStringSmall">Søk DDG eller skriv URL</string>
+    <string name="browser_chrome">Chrome</string>
+    <string name="browser_duckduckgo">DuckDuckGo</string>
+    <string name="browser_firefox">Firefox</string>
+    <string name="ddg_homescreen_label">Søk DDG</string>
+    <string name="help_link">https://duck.co/help/mobile/android</string>
+    <string name="instruction_1_b">Chrome</string>
+    <string name="instruction_2_a">Opne</string>
+    <string name="instruction_bullet_point_1">1</string>
+    <string name="instruction_bullet_point_2">2</string>
+    <string name="instruction_bullet_point_3">3</string>
+    <string name="instruction_done">Ferdig!</string>
+    <string name="add_to">Legg til %s</string>
+    <string name="instruction_1_a">Opne</string>
+    <string name="instruction_firefox_1_a">\@string/instruction_open</string>
```

Comment:
@Freheims could you remove all the "\" before the "@string/" otherwise instead of the reference of the string we show the user just the string name